src/database/migrations/20230609052656-create-user-extension.js

```diff
@@ -58,6 +58,9 @@ module.exports = {
 			deleted_at: {
 				type: Sequelize.DATE,
 			},
+			status: {
```

Comment:
create separate migration file for this refer this https://sequelize.org/docs/v6/other-topics/query-interface/ @Shaikjubedha